Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -125,6 +136,16 @@ namespace Ra
 
     }
 
+    void Gui::MainWindow::update()
+    {
+        //timerData.tasksStart = Core::Timer::Clock::now();

```

Comment:
don't know ;) seems I have trouble with timerData, and when I check in master, there is no timer data in mainwindow anymore ! this pull request is too old and merge do not work well ...